src/components/AppLayout/Sidebar/SafeHeader/index.tsx

```diff
@@ -161,56 +282,55 @@ const SafeHeader = ({
       <Container>
         {/* Identicon */}
         <IdenticonContainer>
-          <FlexSpacer />
           <Threshold />
           <Identicon address={address} size="lg" />
-          <ButtonHelper onClick={onToggleSafeList} data-testid={TOGGLE_SIDEBAR_BTN_TESTID}>
+          <ToggleSafeListButton onClick={onToggleSafeList} data-testid={TOGGLE_SIDEBAR_BTN_TESTID}>
             <StyledIcon size="md" type="circleDropdown" />
-          </ButtonHelper>
+          </ToggleSafeListButton>
         </IdenticonContainer>
 
         {/* SafeInfo */}
-        <StyledTextSafeName size="lg" center>
+        <StyledTextSafeName size="xl" center>
           {safeName}
         </StyledTextSafeName>
         <StyledPrefixedEthHashInfo hash={address} shortenHash={4} textSize="sm" />
         <IconContainer>
           <Track {...OVERVIEW_EVENTS.SHOW_QR}>
-            <ButtonHelper onClick={onReceiveClick}>
+            <StyledQRCodeButton onClick={onReceiveClick}>
               <Icon size="sm" type="qrCode" tooltip="Show QR code" />
-            </ButtonHelper>
+            </StyledQRCodeButton>
           </Track>
           <Track {...OVERVIEW_EVENTS.COPY_ADDRESS}>
-            <CopyToClipboardBtn textToCopy={copyChainPrefix ? `${shortName}:${address}` : `${address}`} />
+            <StyledCopyToClipboardBtn textToCopy={copyChainPrefix ? `${shortName}:${address}` : `${address}`} />
           </Track>
           <Track {...OVERVIEW_EVENTS.OPEN_EXPLORER}>
-            <ExplorerButton explorerUrl={getExplorerInfo(address)} />
+            <StyledExplorerButton explorerUrl={getExplorerInfo(address)} />
           </Track>
         </IconContainer>
 
+        <Paragraph color="black400" noMargin size="md">
+          Total Balance
+        </Paragraph>
+        <StyledText size="xs">{balance}</StyledText>
+        <StyledButton
+          size="md"
+          disabled={!granted}
+          color="primary"
+          variant="contained"
+          onClick={handleNewTransactionClick}
+        >
+          <Text size="xl" color="white">
```

Comment:
The style does not match the Figma file

```
font-weight: 700;
font-size: 14px;
line-height: 20px;
```

Reply:
We decided to go with a lighter style that also matches all other buttons ([Comment in last PR](https://github.com/gnosis/safe-react/pull/3776#issuecomment-1093110790))

Reply:
The link doesn't take me to a comment in that regard but I agree with the justification 👍